### ISSUE TITLE
Peep AI prefers to use the long path instead of the short path to reach a ride.

### ISSUE THREAD

**Issue by Wirlie** [OpenRCT2/OpenRCT2#3449]
**OS:** Windows 10 (x86)
**Version:** 0.0.5
**Commit:** latest commit: cd7482b

- [x] Reproducible in RCT2 (vanilla)? _Yes, I can reproduce this on RCT2 (original bug)_
- [ ] Multiplayer?

**Steps to reproduce:**
I don't know how to reproduce this problem.

**Screenshots / Video:**
The Green line is the path that a guest should take to reach the ride (short). 
The Red line is the wrong path that the guests use to reach the ride (long).
![scr4-2](https://cloud.githubusercontent.com/assets/7508197/14895377/e85fa97c-0d3d-11e6-8f83-501e8ee54c0a.png)

I have tried to reproduce this problem with a mechanic and this does not happen, the mechanic uses the green path (short path) to reach the ride.

**Save game:**
see the guest named "AAAAAAAAAAAAAAA":
[Jardines Gravitatorios.txt](https://github.com/OpenRCT2/OpenRCT2/files/241112/Jardines.Gravitatorios.txt)

**Comment by Nubbie** [comment]
Maybe have something to do with #3423 ?

**Comment by jensj12** [comment]
The problem is not exactly as you described it, my observations:
Put guest AAAAAA next to the entrance and he follows the green path. Same if you the guest between the split and the toilet.
Put the guest beyond the toilet on the red path (between toilet and drink stall) and it'll follow the red path.

I've counted the number of times peeps have a choice on both of the paths (excluding where they merge), on the red path it's 10 and on the green path it's 9. That perfectly explains the behaviour with what I found in #3423, so this is a duplicate.
